admin/updates/update-2-add_gin_index_lowleve_json_data-md5_encoded.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+CREATE INDEX data_metadata_audio_properties_md5_encoded_idx_lowlevel_json ON lowlevel_json USING gin ((data#>'{metadata,audio_properties,md5_encoded}'));
```

Comment:
Could use a line break here before `ON`.

You also should include this into https://github.com/metabrainz/acousticbrainz-server/blob/master/admin/sql/create_indexes.sql.